src/test/feature/query/test-insert.cpp

```diff
@@ -0,0 +1,176 @@
+#include <pwd.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <vector>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <iostream>
+#include <string>
+
+#include "lib/command.h"
+#include "lib/common.h"
+#include "lib/data-gen.h"
```

Comment:
According to this merged [pr](https://github.com/apache/incubator-hawq/pull/683), `#include "lib/data-gen.h"` -> `#include "lib/data_gen.h"`